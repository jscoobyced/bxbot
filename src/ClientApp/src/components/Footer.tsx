import * as React from 'react';

const version = process.env.VERSION;

export class Footer extends React.Component {
    public render() {
        return <footer>${version}</footer>;
    }
}
