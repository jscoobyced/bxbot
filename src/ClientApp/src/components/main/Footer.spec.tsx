import { shallow } from 'enzyme';
import * as React from 'react';
import { Footer } from './Footer';

test('Footer component is unchanged.', () => {
    const footer = shallow(<Footer />);

    expect(footer).toMatchSnapshot();
});