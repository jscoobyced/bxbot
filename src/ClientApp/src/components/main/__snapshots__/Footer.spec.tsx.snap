// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component is unchanged. 1`] = `
<footer>
   - 
</footer>
`;
